registration-processor/pre-processor/registration-processor-external-stage/src/main/java/io/mosip/registrationprocessor/externalstage/stage/ExternalStage.java

```diff
@@ -351,32 +330,29 @@ public void processURL(RoutingContext ctx) {
             fields.add(proofOfExceptionsLabel);
             fields.add(applicantBiometricLabel);
             fields.add(introducerBiometricLabel);
+            fields.add(proofOfConcentLabel);
 
 
             String registrationId = messageDTO.getRid();
             String process = messageDTO.getReg_type().name();
             Map<String, String> docFields = packetManagerService.getFields(registrationId, fields, process, ProviderStageName.PACKET_VALIDATOR);
 
-            if (docFields.get(proofOfAddressLabel) != null) {
-                Document aaa = packetManagerService.getDocument(registrationId, proofOfAddressLabel, process, ProviderStageName.PACKET_VALIDATOR);
-                responceMap.put(proofOfAddressLabel, CryptoUtil.encodeBase64String(aaa.getDocument()));
-            }
-            if (docFields.get(proofOfDateOfBirthLabel) != null) {
+            if (StringUtils.isNotEmpty(docFields.get(proofOfDateOfBirthLabel)) && docFields.get(proofOfDateOfBirthLabel) != null) {

```

Comment:
"docFields.get(proofOfDateOfBirthLabel) != null"  is not necessary since isNotEmpty handles null check internally